go.mod

```diff
@@ -120,6 +120,8 @@ require (
 	sigs.k8s.io/yaml v1.4.0
 )
 
+require github.com/avast/retry-go/v4 v4.6.1
```

Comment:
this require should not be separated. put it in the appropirate existing require section.

also, I think it is overkill to add a rety library that is used once.